### ISSUE TITLE
Ruby 3.0 support

### ISSUE THREAD

**Issue by breisig** [Paxa/fast_excel#72]
Is their plans to release a Ruby 3.0.0 supported version of them gem?

When I try installed 0.3.0, it says

```
fast_excel-0.3.0 requires ruby version ~> 2.0, which is incompatible with the current version, ruby 3.0.0p0
```

**Comment by breisig** [comment]
It looks like using the latest 'master' branch from git allowed it to work. Any idea when their will be a new release?

**Comment by breisig** [comment]
Any idea when the next release will be?

**Comment by breisig** [comment]
?

**Comment by Paxa** [comment]
Released 0.4.0
Sorry for delay
